### ISSUE TITLE
Include the formatted citation in the default Site Report

### ISSUE THREAD

**Issue by clnsmth** [PASTAplus/dashboard#22]
At the May LTER Water Cooler on EDI Reporting Tools, @seagrinch requested the default Site Report include the formatted citation rather than offering it as a separate table. Doing so would streamline use. 

Currently, to combine and access this information, the user must generate two site reports (the default report and one with the formatted citation) then join these tables on the "Package ID" field.

The proposed change: https://docs.google.com/spreadsheets/d/13oMlFiQivWadzli8X8cCx47Jne3qXTwOh5sPoevL-tY/edit#gid=0

Does this look right to you @seagrinch?

_Note: The current implementation posts a warning about the formatted citation option being slower. Maybe this is slow enough that the proposed change here is a no-op._

**Comment by seagrinch** [comment]
Yes that looks good.  If you wanted to provide more options, you could provide details-only, citation-only and combined, with the caveat being the 2nd and 3rd options might be slower.

The citation report for PAL shows up in ~2-3 seconds, so the overhead isn't huge for a small site scope.  But neither report currently works for edi, so I'm not sure what that means.  Pagination is a good way to speed up web page access, and folks are willing to wait longer for downloads.

**Comment by clnsmth** [comment]
Thanks @seagrinch. We'll take these comments into consideration.
